iceoryx_hoofs/test/moduletests/test_cxx_string.cpp

```diff
@@ -77,11 +77,11 @@ TYPED_TEST(stringTyped_test, CopyConstructStringOfSizeCapaResultsInSizeCapa)
     using MyString = typename TestFixture::stringType;
     constexpr auto STRINGCAP = MyString::capacity();
     std::string testString(STRINGCAP, 'M');
```

Comment:
For these cases I would prefer to add such a constructor to `iox::string(uint64_t count, char ch` like `std::string` has to replace the `unsafe_assign`s in the tests. What do you think?

Reply:
@elBoberido You are right, a quick search revealed: #1431 Closing this thread then.